assets.src/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "redmine-tracky",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
what was the reason to delete the package.lock.json? the reason to have a package-lock is to freeze the dependency version and not break it randomly at a build. that's why we have that